queries/usersQueries.js

```diff
@@ -1,16 +1,23 @@
-const usersData = require('../usersData.json');
+const db = require('../db/dbConfig.')
 
-const idToUserIndex = usersData.reduce((index, user) => {
-  const { id } = user;
-  index[id] = user;
-  return index;
-}, {});
+const getAllUsers = async () => {
+  try {
+    const allUsers = await db.any(`SELECT * from users`);
+    return allUsers;
+  } catch (err) {
```

Comment:
Just to elaborate on the use of try/catch here. This can actually be an issue because now your controller code will not know if `getAllUsers()` is returning it an array of users or an error. This means you controller code will need to check what it got from the query: is it a correct response or an error? In some ways it might be simpler just to allow the query method to throw an error and use try/catch only in the controller.